index.md

```diff
@@ -1,113 +1,253 @@
+          <i class="fa fa-github"></i>
+        </a>
+      </div>
+      @ethereumjs/trie
+    </div>
+    <div class="repo-description">
+      Implementation of a Merkle Patricia Tree.
+    </div>
+  </div>
+
+  <div class="repo-box">
+    <div class="repo-header">
+      <div class="repo-header-right">
+        <a href="https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/rlp" alt="GitHub URL">
+          <i class="fa fa-github"></i>
+        </a>
+      </div>
+      @ethereumjs/rlp
+    </div>
+    <div class="repo-description">
+      RLP encoding and decoding.
+    </div>
+  </div>
+
+  <div class="repo-box">
+    <div class="repo-header">
+      <div class="repo-header-right">
+        <a href="https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/util" alt="GitHub URL">
+          <i class="fa fa-github"></i>
+        </a>
+      </div>
+      @ethereumjs/util
+    </div>
+    <div class="repo-description">
+      Utilities for 4844, bytes, signatures, withdrawals, addresses and other.
+    </div>
+  </div>
+
+  <div class="repo-box">
+    <div class="repo-header">
+      <div class="repo-header-right">
+        <a href="https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/wallet" alt="GitHub URL">
+          <i class="fa fa-github"></i>
+        </a>
+      </div>
+      @ethereumjs/wallet
+    </div>
+    <div class="repo-description">
+      Wallet implementation with key management and various import formats.
+    </div>
+  </div>
+</div>
 
-You will need to interface with the Ethereum network. [web3.js](https://github.com/ethereum/web3.js) and [ethers.js](https://github.com/ethers-io/ethers.js) both provide a complete Javascript API to interact with the RPC interface. If looking for a more lightweight option, [ethereumjs-abi](https://github.com/ethereumjs/ethereumjs-abi) or [ethjs-abi](https://github.com/ethjs/ethjs-abi) can handle the ABI encoding.
+<div class="separator"></div>
 
-### Build for the Web?
 
-Most of the EthereumJS libraries can be transpiled with [babel](https://babeljs.io/) to be used in web context.
-We provide [browser-builds](https://github.com/ethereumjs/browser-builds) for some EthereumJS libraries, be
-warned though that these might often be slightly out of date.
+<div class="repo-group">
+  <h3><i class="fa fa-globe"></i> Network</h3>
+  <p>Implementation of the Ethereum network communication layer.</p>
+</div>
 
-# Projects not Tracked Here
+<div class="repo-group">
+  <div class="repo-box">
+    <div class="repo-header">
+      <div class="repo-header-right">
+        <a href="https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/devp2p" alt="GitHub URL">
+          <i class="fa fa-github"></i>
+        </a>
+      </div>
+      @ethereumjs/devp2p
+    </div>
+    <div class="repo-description">
+      Devp2p implementation with support for Discovery (DPT), RLPx transport, ETH protocol and SNAP.
```

Comment:
Can we mention LES here as well?

Reply:
I think LES is somewhat deprecated now, right?

Reply:
Yes, it's in some weird hybrid state I guess, I think it does not make much different if we add or not, will add then